### PR TITLE
UTF-8 error fix in excluded path

### PR DESCRIPTION
**Pull request by kugan49**

### CODE REVIEW

**Comment by kugan49** [comment]
All is in title

File: settings.py [hunk 1]
```diff
@@ -76,7 +76,11 @@ def get_setting(setting):
     elif setting in strings:
         return str(utils.ADDON.getSetting(setting))
     elif setting in paths:
-        return translatePath(utils.ADDON.getSetting(setting).encode("utf-8"))
+        try: 
+            path = utils.ADDON.getSetting(setting).encode("utf-8")
+        except UnicodeDecodeError:
```
Comment (Anthirian) [inline settings.py]: I'm not really sure why we need to catch this type of exception here. Any path that contains unicode characters will be encoded in UTF-8 already with my code, and any path that does not contain unicode will not behave any differently to if it wasn't encoded in UTF-8. I think my code already covered both cases and there is no need to catch exceptions here.

Comment (kugan49) [inline settings.py]: in fact I had errors with "é" caracter in exclusion path.
Adding this exception make the job in this case

Comment (Anthirian) [inline settings.py]: Oh right, my mistake. I was able to reproduce the problem.

**Comment by Anthirian** [comment]
Thanks!

**Pull request merged**